Xamarin.PropertyEditing.Mac/Controls/PropertyEditorControl.cs

```diff
@@ -30,6 +30,7 @@ public IHostResourceProvider HostResources
 
 		public const int DefaultControlHeight = 24;
 		public const int DefaultFontSize = 11;
+		public const int DefaultMenuFontSize = DefaultFontSize + 1;
```

Comment:
We're already trying to get away from a "DefaultFontSize" by using the mac named sizes, let's not compound the problem.

Reply:
Removed.